jsapp/js/components/common/icon.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {ComponentStory, ComponentMeta} from '@storybook/react';
+import Icon from 'js/components/common/icon';
+import {IconNames} from 'jsapp/fonts/k-icons';
+
+export default {
+  title: 'common/icon',
+  component: Icon,
+  argTypes: {},
+} as ComponentMeta<typeof Icon>;
+
+const Template: ComponentStory<typeof Icon> = (args) => <Icon {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {};
+
+export const AllIcons = () => (
```

Comment:
I like this list. I think we can delete `/jsapp/k-icons-html-template.hbs` (and the mention in `/scripts/generate_icons.js`)

Reply:
Ok, removed it and the generate icons script still seems to run.